projects/common/src/navigation/navigation.service.ts

```diff
@@ -272,6 +272,10 @@ export class NavigationService {
     return this.platformLocation.href;
   }
 
+  public getRootPathUrl(): string {
+    return window.location.host;
```

Comment:
please refer to L271

Reply:
what you need will be in `origin` in `platformLocation`

Reply:
platformLocation does not provide host, through that we'll need to add hostname and port, is that fine?

Reply:
updated